Fix:
conftest.c:41:1: error: implicit declaration of function 'puts' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://salsa.debian.org/clint/fakeroot/-/merge_requests/15
--- configure.ac.orig	2022-01-17 14:40:08.000000000 -0600
+++ configure.ac	2022-03-04 07:22:45.000000000 -0600
@@ -129,6 +129,7 @@
   for second in void char; do
     for third in size_t int; do
       AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <unistd.h>
+#include <stdio.h>
         $zeroth readlink(const char *path, $second *buf, $third bufsiz);]], [[puts("hello, world");]])],
 	[readlink_retval=$zeroth
 	 readlink_buf_arg=$second
